models/Movie.model.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose')
+const movieSchema = new mongoose.Schema({
+    title:{
+        type:String,
+        required: true,
+    },
+    genre:{
+        type:String,
+        required: true,
+    },
+    plot:{
+        type:String,
+        required: true,
+    },
+    cast:{
```

Comment:
The problem is that is expecting just one value instead of an array of values :) 
You also needed to reference the Model for the relationship 

```suggestion
     cast:[{
        type: mongoose.Schema.Types.ObjectId,
        ref: 'Celebrity'
        required: true,
    }],
```